tests/candev/main.c

```diff
@@ -23,19 +23,33 @@
 
 #include <debug.h>
 #include <errno.h>
+#include <isrpipe.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <isrpipe.h>
 #include "shell.h"
 #include "can/device.h"
 
-#ifdef BOARD_NATIVE
+#if defined(BOARD_NATIVE)
 
 #include <candev_linux.h>
-
 static candev_linux_t linux_dev;
 
+#elif defined(CAN_DRIVER_MCP2515)
```

Comment:
can-frames with an extended-ID are not received when the mcp2515 driver is used.